forgettable-frontend/src/components/EncounterCard/helper.js

```diff
@@ -0,0 +1,28 @@
+export function stringToColor(string) {
```

Comment:
Very nice file! I think this will be needed in more than just the encounter card component - would you be able to put this under src/functions?

Reply:
Sure will do!